files/fr/web/api/css_properties_and_values_api/index.html

```diff
@@ -0,0 +1,52 @@
+---
+title: CSS Properties and Values API
+slug: Web/API/CSS_Properties_and_Values_API
+translation_of: 'Web/API/CSS_Properties_and_Values_API'
+---
+<p>{{SeeCompatTable}} L'<strong>API Propriétés et valeurs CSS</strong> — qui fait partie de l'ensemble d'API <a href="/en-US/docs/Web/Houdini">CSS Houdini</a> — sert aux développeurs à définir explicitement leurs {{cssxref('--*', 'propriétés CSS personnalisées')}}, ce qui permet la vérification des types de propriétés, de leur donner des valeurs par défaut et de déterminer si une propriété peut hériter ses valeurs ou non.</p>
```

Comment:
Well, I thought cssxref could take parameters to avoid display stuff in monospaced/code but after checking https://github.com/mdn/yari/blob/main/kumascript/macros/cssxref.ejs I updated my understanding of it.
I think a simple <a href...> link would be better here than the cssxref macro.

Reply:
This comment can be applied for other occurrences as well.

Reply:
(and yeah, I've seen the English page :x )

Reply:
Makes sense to me. Fixed in https://github.com/mdn/translated-content/pull/1765/commits/0fca36b8d0da908a0d704478ec7a60bf13bdbb51